doc/ug/dv_methodology.md

```diff
@@ -415,6 +415,36 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
+### X-Propagation (Xprop)
```

Comment:
nit: 'P' probably does not need to be capitalized (here and below).

Reply:
use 'P' as most of places use Cap P